hkm/templates/hkm/snippets/navigation.html

```diff
@@ -49,7 +49,7 @@
       {% if not user.profile.is_museum %}
       {% if user.is_authenticated %}
       <li class="list-item_user">
-        <a id="profile-dropdown" class="nav-user" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+        <a id="profile-dropdown" aria-label="{% trans 'Profile menu' %}" class="nav-user" href="#" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
           <span>{{ user }}</span>
           <img class="nav-svg" src="/static/hkm/svg/Person.svg" alt="person-svg" >
```

Comment:
This alt text doesn't offer much use. This may not be the correct place, but you could mark it as decorative by applying an empty alt text.

Reply:
Marked as decorative.